internal/backend/crypto/gpg/colons/parse_colons.go

```diff
@@ -10,11 +10,11 @@ import (
 )
 
 var (
-	// John Doe (user) <john.doe@example.com>
+	// John Doe (user) <john.doe@example.com>.
```

Comment:
The regex isn't matching the point, not sure we should have one there. 
Same for the two other comment below.
Maybe add a //nolint:godot for that part?